graphics/poppler/Portfile

```diff
@@ -56,16 +56,14 @@ configure.cppflags-delete -I${prefix}/include
 configure.ldflags-append -liconv
 gobject_introspection yes
 
-compiler.cxx_standard 2014
-compiler.c_standard 1999
+compiler.cxx_standard 2017
+compiler.c_standard 2001
```

Comment:
Is this supposed to be `2011`? AFAIK there is no C 2001.

Reply:
Correct! My bad ... 7e30f6e6d8376b24530a6d052fbfde11ddb8473a